raiden/network/rpc/smartcontract_proxy.py

```diff
@@ -77,6 +80,8 @@ def transact(self, function_name: str, *args, **kargs):
             action = inspect_client_error(e, self.jsonrpc_client.eth_node)
             if action == ClientErrorInspectResult.INSUFFICIENT_FUNDS:
                 raise InsufficientFunds('Insufficient ETH for transaction')
+            elif action == ClientErrorInspectResult.TRANSACTION_UNDERPRICED:
+                raise ReplacementTransactionUnderpriced()
```

Comment:
Put a meaningful error in the constructor here in case we catch it somewhere else in the future and want to look at the error message.